x/distribution/types/errors.go

```diff
@@ -17,4 +17,5 @@ var (
 	ErrNoValidatorExists       = errors.Register(ModuleName, 12, "validator does not exist")
 	ErrNoDelegationExists      = errors.Register(ModuleName, 13, "delegation does not exist")
 	ErrInvalidProposalContent  = errors.Register(ModuleName, 14, "invalid proposal content")
+	ErrInvalidSigner           = errors.Register(ModuleName, 15, "expected authority account as only signer for proposal message")
```

Comment:
Some existing `ErrInvalidSigner` have a different message, could we standardize them?